### PR TITLE
tpm_renders.md: Initial commit

### PR DESCRIPTION
**Pull request by piotrdanio**
Tpm renders chapter after a few correct

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ nav:
     - 'Target TPM hardware interfaces': hardware_interfaces.md
     - 'Current issues': issues.md
     - 'TPM modules pinout': tpm_modules_pinout.md
```
Comment (acwiek) [inline mkdocs.yml:24]: Group tpm_modules_pinout.md and tpm_renders.md to - 'lpnTPM mechanical design'

File: docs/tpm_renders.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+## The TPM concept
+According to the many varied pinouts necessary was design the concept of TPM 
+module which can fit the motherboard pinout. There was the consideration of 
+two kinds of the interface (SPI and LPC). The whole setup would be made of two 
+PCBs. One is the main PCB with the MCU and SPI or LPC configured socket and the 
+second is the adapter that fits the motherboards pinouts.
+![Flash](images/renders/koncepcja_TPM_adapter.svg)
+
+## The adapter
```
Comment (acwiek) [inline docs/tpm_renders.md:9]: I don't think this chapter is still relevant. Below in the '## Concept of slim adapter' part of the document, you present newer version of adapter `### The adapter:
<center>Flash</center>`

**Comment by acwiek** [comment]
docs/images/renders/koncepcja_TPM_adapter.svg  - are you using this file somewhere in the documentation?
 please keep filenames in English

File: docs/tpm_renders.md (hunk 2 above)
Comment (piotrdanio) [reply docs/tpm_renders.md]: ok

File: mkdocs.yml (hunk 1 above)
Comment (piotrdanio) [reply mkdocs.yml]: ok

**Comment by acwiek** [comment]
please keep filenames (images) in one naming convetion - probably snake_case in this case.

File: mkdocs.yml [hunk 3]
```diff
@@ -19,11 +19,14 @@ nav:
...
+      - 'TPM modules pinout': tpm_modules_pinout.md
+      - 'TPM modules renders': tpm_renders.md
     - 'Current issues': issues.md
-    - 'TPM modules pinout': tpm_modules_pinout.md
     - 'Journal': journal.md
-    - 'RaspberryPi demonstration with TPM module':
-      - 'Connecting Raspberry Pi with Windows OS': rpi_connection.md
-      - 'Running TPM 2.0 module on Raspberry Pi': rpi.md
-      - 'FAQ': rpi_faq.md
+
```
Comment (acwiek) [inline mkdocs.yml:32]: remove empty line

File: docs/tpm_renders.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+## The TPM concept
+According to the many varied pinouts necessary was design the concept of TPM 
+module which can fit the motherboard pinout. There was the consideration of 
+two kinds of the interface (SPI and LPC). The whole setup would be made of two 
+PCBs. One is the main PCB with the MCU and SPI or LPC configured socket and the 
+second is the adapter that fits the motherboards pinouts.
+![Flash](images/renders/concept_tpm.svg)
+
```
Comment (acwiek) [inline docs/tpm_renders.md:8]: please remove the empty line. leave only one

File: mkdocs.yml (hunk 3 above)
Comment (ogtomi) [reply mkdocs.yml]: Empty line removed in commit `d6c9c0644891819aae6b6107230aead4950a9d20`

File: docs/tpm_renders.md (hunk 4 above)
Comment (ogtomi) [reply docs/tpm_renders.md:8]: Empty line removed in  commit `28fb95abb7080013df7b8d30a6991f7dfabd18a4` . This commit also deletes html tags used for images.

**Pull request merged**